src/til/ut_til/SizeTests.cpp

```diff
@@ -303,7 +303,7 @@ class SizeTests
         Log::Comment(L"1.) Scale results in value that is too large.");
         {
             const til::size sz{ 5, 10 };
-            constexpr auto scale = std::numeric_limits<float>().max();
+            constexpr auto scale = 1e12f;

```

Comment:
Same deal here: https://godbolt.org/z/6M6Mcjdx9

Reply:
This test is designed to force an overflow. Shouldn't we _use_ `max`?

Reply:
It's still overflowing a `int32_t`, just not by as much as before. 😅